order/supplier/receive/README.md

```diff
@@ -140,12 +156,36 @@ The `mergedItemDetails` will contain the original item details added by the buye
 * `serialNumber`: is an unique identifier assigned incrementally or sequentially to an item, to uniquely identify it.
 * `batchNumber`: is an identification number assigned to a particular quantity or lot of material from a single manufacturer
 
-### Requested planned delivery schedule
+## Native Delivery schedule
 
-`lines.deliverySchedule`: the requested delivery schedule by the buyer.
-* `position`: the optional position in the delivery schedule. Not to be confused with the `line.position`
-* `date`: the requested delivery date of this delivery schedule position. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
-* `quantity`: the requested quantity of this delivery schedule position. Quantity has a decimal `1234.56` format with any number of digits.
+`lines.deliverySchedule`: the current delivery schedule, either `Issued` or `Confirmed`.
+
+`lines.deliveryScheduleIncludingRequests`: the current planned delivery schedule, either `Issued`, `In Progress` (having an open `Proposal` or `Reopen Request`) or `Confirmed`. This field is only supported as native delivery schedule.
```

Comment:
This field is also used when using the Simple Delivery Schedule

Reply:
There is no `deliveryScheduleIncludingRequests` or `scheduledDeliveryIncludingRequests` when using the simple delivery schedule.